Fmaraev/Hometask_2/stepik_org.py

```diff
@@ -0,0 +1,233 @@
+"""Из передачи “Здоровье” Аня узнала, что рекомендуется спать
+хотя бы A часов в сутки, но пересыпать тоже вредно и не стоит
+спать более B часов. Сейчас Аня спит H часов в сутки.
+Если режим сна Ани удовлетворяет рекомендациям передачи
+“Здоровье”, выведите “Это нормально”. Если Аня спит менее
+A часов, выведите “Недосып”, если же более B часов, то выведите “Пересып”."""
+a = int(input())
+b = int(input())
+h = int(input())
+if h < a:
+    print('Недосып')
+elif a <= h <= b:
+    print('Это нормально')
+else:
+    print('Пересып')
+
+"""Напишите простой калькулятор, который считывает с пользовательского
+ввода три строки: первое число, второе число и операцию, после чего
+применяет операцию к введённым числам ("первое число" "операция"
+"второе число") и выводит результат на экран.
+Поддерживаемые операции: +, -, /, *, mod, pow, div, где
+mod — это взятие остатка от деления,
+pow — возведение в степень,
+div — целочисленное деление."""
+a = float(input())
+b = float(input())
+q = str(input())
+if q == '+':
+    print(a + b)
+elif q == '-':
+    print(a - b)
+elif q == '*':
+    print(a * b)
+elif q == '/' and b == 0:
+    print('Деление на 0!')
+elif q == '/':
+    print(a / b)
+elif q == 'mod' and b == 0:
+    print('Деление на 0!')
+elif q == 'mod':
+    print(a % b)
+elif q == 'pow':
+    print(a ** b)
+elif q == 'div' and b == 0:
+    print('Деление на 0!')
+elif q == 'div':
+    print(a // b)
+
+"""Напишите программу, которая получает на вход
+ ри целых числа, по одному числу в строке, и выводит
+на консоль в три строки сначала максимальное, потом
+минимальное, после чего оставшееся число.
+На ввод могут подаваться и повторяющиеся числа."""
+a = int(input())
+b = int(input())
+c = int(input())
+if a >= b and a >= c:
+    mx = a
+elif b >= a and b >= c:
+    mx = b
+else:
+    mx = c
+if a <= b and a <= c:
+    mn = a
+elif b <= a and b <= c:
+    mn = b
+else:
+    mn = c
+if mx == a and mn == b or mn == a and mx == b:
+    md = c
+elif mx == b and mn == c or mn == b and mx == c:
+    md = a
+else:
+    md = b
+print(mx)
+print(mn)
+print(md)
+
+"""Напишите программу, которая считывает со стандартного ввода
+целые числа, по одному числу в строке, и после первого введенного
+нуля выводит сумму полученных на вход чисел."""
+s = 0
+a = int(input())
+if a == 0:
+        print(s)
+while a != 0:
```

Comment:
```suggestion
a = 0
while 1:
    a = int(input())
    if a == 0:
         print(s)
         break
    else:
        s +=a
```
and we don't need previous if